src/api/support/support.v6.service.js

```diff
@@ -24,7 +24,7 @@ angular.module('ovh-api-services').service('OvhApiSupportV6', ($resource, $cache
     create: {
       interceptor,
       method: 'POST',
-      url: '/support/tickets/:id/create',
+      url: '/support/tickets',
```

Comment:
We should keep the previous path as well as "One shot ticketing" may not be prodded in the US :)

Reply:
created 'postTicket' method in order to keep the 'create' one too